akka-docs/src/main/paradox/typed/persistence-style-durable-state.md

```diff
@@ -0,0 +1,32 @@
+# Style Guide 
+
+@@@ div { .group-scala }
+## Command handlers in the state
+
+We can take the previous bank account example one step further by handling the commands in the state too.
+
+Scala
+:  @@snip [AccountExampleWithCommandHandlersInDurableState.scala](/akka-cluster-sharding-typed/src/test/scala/docs/akka/cluster/sharding/typed/AccountExampleWithCommandHandlersInDurableState.scala) { #account-entity }
+
+Notice how the command handler is delegating to `applyCommand` in the `Account` (state), which is implemented
+in the concrete `EmptyAccount`, `OpenedAccount`, and `ClosedAccount`.
+
+@@@
+
+## Optional initial state
+
+Sometimes it's not desirable to use a separate state class for the empty initial state, but rather treat that as
+there is no state yet.
+@java[`null` can then be used as the `emptyState`, but be aware of that the `state` parameter
```

Comment:
```suggestion
@java[You can use `null` can as the `emptyState`, but be aware of that the `state` parameter
```

Reply:
I think the second 'can' is unneeded - (after 'null')
@java[You can use `null` as the `emptyState`, but be aware of that the `state` parameter

Reply:
done ..